Makefile

```diff
@@ -45,8 +65,8 @@ start-openshift-with-catalog:
 
 	@echo "---- Granting admin rights to Developer ----"
 	oc login -u system:admin
-	oc adm policy add-cluster-role-to-user cluster-admin developer
-	oc login -u developer -p developer
+	oc adm policy add-cluster-role-to-user cluster-admin $(_USERNAME)
```

Comment:
This probably needs to go away for the remove cluster.

Reply:
Hmm, this one is not called for the remote container. What do you mean? Should I add it there?

Reply:
ooooh right... Sorry Martin... I'm blind... and need coffee ;)